packages/truffle-box/test/sandbox.js

```diff
@@ -0,0 +1,24 @@
+const { sandbox } = require("../");
+const assert = require("assert");
+
+describe("Box.sandbox", () => {
+  let options;
+
+  it("successfully unboxes and returns TruffleConfig when passed a valid options object", async () => {
+    options = { name: "default", force: true };
+    try {
+      const config = await sandbox(options);
+      assert(config);
+      assert(config.truffle_directory);
+    } catch (error) {
+      throw error;
```

Comment:
Why do we need the `try/catch`?

Reply:
Thanks for reviewing and being so thorough!